scripts/post-uninstall-rpm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This check works only for RPMs
+if [ $1 -ge 1 ]; then
+    # Package upgrade, do not uninstall
+    exit 0
+fi
+
+# Delete the link to the binary
+if type update-alternatives >/dev/null 2>&1; then
+    update-alternatives --remove "httptoolkit" "/opt/HTTP Toolkit/httptoolkit"
```

Comment:
As explained [here](https://github.com/IsmaelMartinez/teams-for-linux/issues/958#issue-1904907507), the second argument to `update-alternatives --remove` should be `/opt/HTTP Toolkit/toolkit`, not `/usr/bin/toolkit`.